### PR TITLE
feat: add projct type count analytics

### PR DESCRIPTION
**Pull request by YairZ101**
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules

#### What does this PR do?

Adds new analytics for project type counts

### CODE REVIEW

File: src/cli/commands/test/iac-local-execution/analytics.ts [hunk 1]
```diff
@@ -10,21 +10,23 @@ export function addIacAnalytics(
...
 
   formattedResults.forEach((res) => {
     totalIssuesCount =
       (totalIssuesCount || 0) + res.result.cloudConfigResults.length;
 
-    const packageManagerConfig = res.packageManager;
-    packageManagers.push(packageManagerConfig);
+    const projectType = res.packageManager;
+    packageManagers.push(projectType);
+    projectTypeAnalytics[projectType] = projectTypeAnalytics[projectType] ?? {
```
Comment (teodora-sandu) [inline src/cli/commands/test/iac-local-execution/analytics.ts:22]: Screenshot 2021-12-02 at 09 45 03

I think here you don't need the nested `projectType`

Comment (YairZ101) [reply src/cli/commands/test/iac-local-execution/analytics.ts:22]: yea you are correct, thx for catching that.

Comment (teodora-sandu) [reply src/cli/commands/test/iac-local-execution/analytics.ts:22]: Looks great now
Screenshot 2021-12-02 at 10 00 58

**Review by teodora-sandu — APPROVED** [review]
🎉

**Pull request merged**